src/main/kotlin/com/intershop/gradle/icm/ICMProjectPlugin.kt

```diff
@@ -71,12 +71,6 @@ open class ICMProjectPlugin @Inject constructor(private var projectLayout: Proje
 
             pluginConfig.createLibFilterFile()
             configureProjectTasks(this, pluginConfig)
-
-            if(extension.projectConfig.newBaseProject.get()) {
-                configureBasePublishingTasks(this, extension)
```

Comment:
Why didn't you delete method `configureBasePublishingTasks` also?

Reply:
cleaned up the class